compiler/noirc_driver/src/lib.rs

```diff
@@ -102,6 +102,11 @@ pub struct CompileOptions {
     /// Enable the experimental elaborator pass
     #[arg(long, hide = true)]
     pub use_elaborator: bool,
+
+    /// Enable printing results of comptime evaluation: provide a path suffix
+    /// for the module to debug, e.g. "package_name/src/main.nr"
+    #[arg(long)]
+    pub debug_comptime_scope: Option<String>,
```

Comment:
What does the "_scope" suffix on this name refer to?
If I had to guess it'd be the file we're passing in but it reads oddly to me.

Reply:
Yes, it's the file/suffix that we're looking at. `scope` was picked as an abbreviation to appx. "file suffix to filter by," but I'm open to other name suggestions

Reply:
Maybe "debug_comptime_in_file"